ephios/core/templates/core/disposition/fragment_participation.html

```diff
@@ -8,6 +8,13 @@ <h6 class="mb-0">
             <div class="col-10 pr-3">
                 {{ form.instance.participant.icon }}
                 {{ form.instance.participant }}
+                {% with form.instance.participant|conflicting_shifts:shift as conflicting_shifts %}
+                    {% if conflicting_shifts %}
+                        <span class="fa fa-exclamation-triangle text-danger"
+                              data-toggle="tooltip"
+                              title="{{ form.instance.participant }} already participates in {{ conflicting_shifts|join:", " }} at the same time."></span>
+                    {% endif %}
+                {% endwith %}
```

Comment:
You should probably call `handleForms` in the `disposition.js` when adding new forms to get the tooltip displayed correctly.